pkg/rtc/mediatrackreceiver.go

```diff
@@ -182,11 +182,6 @@ func (t *MediaTrackReceiver) SetupReceiver(receiver sfu.TrackReceiver, priority
 		if priority == 0 {
 			t.trackInfo.MimeType = receiver.Codec().MimeType
 			t.trackInfo.Mid = mid
-
-			// for clients don't have simulcast codecs (old version or single codec), add the primary codec
-			if len(t.trackInfo.Codecs) == 0 && t.trackInfo.Type == livekit.TrackType_VIDEO {
-				t.trackInfo.Codecs = append(t.trackInfo.Codecs, &livekit.SimulcastCodecInfo{})
-			}
```

Comment:
This is where a codec was added to handle older clients. This happens when `OnTrack` happens. Moving this when pending track is set up.